nets/DevNetSpec.ts

```diff
@@ -118,7 +118,7 @@ export async function spawnDevNet({
       chainSpecPath,
       "--port",
       `${httpPort}`,
-      "--ws-port",
+      await getBinaryWsPortCliArg(binary),
```

Comment:
Can await this within a `Promise.all` above (should perhaps also contain the two `getFreePort` calls)